src/m365/planner/commands/task/task-get.ts

```diff
@@ -0,0 +1,57 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Task } from '../../Task';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+}
+
+class PlannerTaskGetCommand extends GraphItemsListCommand<Task> {
+  public get name(): string {
+    return commands.TASK_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieve the details of the specified planner task';
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/planner/tasks/${encodeURIComponent(args.options.id as string)}/details`,
```

Comment:
It occurred to me just now: since we're calling the API to get task details, shouldn't this command be named `planner task details get` instead of `planner task get` which would call `${this.resource}/v1.0/planner/tasks/${encodeURIComponent(args.options.id)}` and retrieve all task information?

Reply:
Totally right; created a new issue for it #3019 and will rename this PR